src/main/scala/subsystem/HasTiles.scala

```diff
@@ -229,7 +229,8 @@ trait CanAttachTile {
 
   /** Narrow waist through which all tiles are intended to pass while being instantiated. */
   def instantiate(implicit p: Parameters): TilePRCIDomain[TileType] = {
-    val tile_prci_domain = LazyModule(new TilePRCIDomain[TileType](tileParams.hartId) {
+    val clockSinkParams = tileParams.clockSinkParams.copy(name = Some(s"${tileParams.name.getOrElse("")}_${tileParams.hartId}"))
```

Comment:
for backwards compatibility's sake, can you make this `tileParams.name.getOrElse("core")`?

Reply:
Ok